integrations/telegram/src/index.ts

```diff
@@ -177,6 +177,9 @@ const integration = new bp.Integration({
     if (userFieldsToUpdate.pictureUrl || userFieldsToUpdate.name) {
       await client.updateUser({
         ...user,
+        tags: {
+          id: user.tags.id,
```

Comment:
**bug:** This code tried setting tags that do not belong to the telegram integration